client/layout/sidebar/item.jsx

```diff
@@ -37,31 +36,26 @@ export default React.createClass( {
 	},
 
 	render() {
-		const { className, children, icon, isPlaceholder, label, link,
-			onNavigate, postType, selected, tipTarget } = this.props;
```

Comment:
Just curious why you removed this destructuring line in favor of repeating `this.props` several times below?